app/api/routes.py

```
@@ -278,6 +280,31 @@ def get_record(
+):
+    try:
+        avg_data, start_date, end_date = recordService.get_weekly_average_records(db, current_user.id, target_date)
+
+        # Pydantic 모델로 변환하여 응답
+        return WeeklyAverageResponse(
+            start_date=start_date,
+            end_date=end_date,
+            data=WeeklyAverageData(**avg_data)
+        )
+
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)